client/components/date-range/shortcuts.tsx

```diff
@@ -0,0 +1,103 @@
+import { Button } from '@wordpress/components';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+
+const DATERANGE_PERIOD = {
+	DAY: 'day',
+	WEEK: 'week',
+	MONTH: 'month',
+};
+
+const DateRangePickerShortcuts = ( {
+	currentShortcut,
+	onClick,
+}: {
+	currentShortcut?: string;
+	onClick: ( shortcut: {
```

Comment:
The definition here doesn't match `handleShortcutClick` function in the date range component.